Venmo/src/main/java/com/braintreepayments/api/venmo/VenmoPaymentAuthResultInfo.kt

```diff
@@ -0,0 +1,13 @@
+package com.braintreepayments.api.venmo
+
+import com.braintreepayments.api.BrowserSwitchFinalResult
+
+/**
+ * Details of a [VenmoPaymentAuthResult.Success]
+ */
+class VenmoPaymentAuthResultInfo internal constructor
+    (private val browserSwitchSuccess: BrowserSwitchFinalResult.Success) {
```

Comment:
Can we make this a public property and remove the getter?

Reply:
I think we want this to be internal - merchants shouldn't need to access it. Removed the separate getter and used the restrict annotation though. Feel free to change the way this is set up to align with other modules in the alignment ticket work